tests/e2e/reconcile/04-check-bucket-name.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # workaround for asserting parts of the configmap
+  # can be removed once https://github.com/kudobuilder/kuttl/issues/262 is implemented
+  - command: "/bin/sh -c 'kubectl get --namespace $NAMESPACE configmap tempo-simplest -o yaml | grep \"bucket: tempo-updated\"'"
```

Comment:
Alternatively we could assert the entire configmap here, but then this tests needs updating for every (unrelated) change to the configmap.